backend/src/test/kotlin/org/loculus/backend/controller/submission/GetOriginalMetadataEndpointTest.kt

```diff
@@ -189,4 +188,14 @@ class GetOriginalMetadataEndpointTest(
         submissionControllerClient.getOriginalMetadata()
             .andExpect(status().isOk)
     }
+
+    // Regression test for https://github.com/loculus-project/loculus/issues/4036
```

Comment:
```suggestion
```

I'd actually prefer not to document this (the test generally makes sense in my opinion, no need to explain the reason).

Reply:
Hmm, why not explain the reason? It's useful context, established projects I've seen often link to issues for further context, e.g. https://github.com/rust-lang/rust-clippy/blob/master/tests/ui/cast_alignment.rs#L45

Reply:
In this case, I don't feel that it is very useful information. If one would like to learn about the background/history of a change, one can look up the git history. Also, it's not something that we've been doing otherwise and it seems inconsistent. But it's not a big deal, feel free to keep it if you like :)